build.sbt

```diff
@@ -9,7 +9,7 @@ Global / onChangedBuildSource := ReloadOnSourceChanges
 Global / resolvers += "scala-integration" at
   "https://scala-ci.typesafe.com/artifactory/scala-integration/"
 
-def localSnapshotVersion = "1.3.1-SNAPSHOT"
+def localSnapshotVersion = "1.3.2-SNAPSHOT"
```

Comment:
mima?

Reply:
What do you mean?

Reply:
Ach, you mean mimaPreviousArtifacts? I am thinking we shouldn't actually update it if it works.